lib/SPIRVProducerPass.cpp

```diff
@@ -2685,12 +2685,6 @@ void SPIRVProducerPass::GenerateResourceVars(Module &) {
                              new SPIRVInstruction(spv::OpDecorate, Ops));
       }
       break;
-    case clspv::ArgKind::ReadOnlyImage:
```

Comment:
Because we generate read_only images as sampled I had to remove this to validate.